### ISSUE TITLE
Convert ~ to $HOME in stdin

### ISSUE THREAD

**Issue by duraki** [BishopFox/sliver#504]
When I use `generate` command with `--file` param something similar to `~/file` the tilde (`~`) is being taken literally. It should be refixed with $HOME internally. For absolute tilde, one can use `"<~quotemarks>"` without parenthesis.

Example:

```
sliver > version

[*] Client v1.4.20 - 7a38ddffb60d9205b1cef1bac430af742eff4e99 - darwin/amd64
    Compiled at 2021-07-23 18:37:13 +0200 CEST
    Compiled with go version go1.16.6 darwin/amd64


[*] Server v1.4.20 - 7a38ddffb60d9205b1cef1bac430af742eff4e99 - darwin/amd64
    Compiled at 2021-07-23 18:37:13 +0200 CEST

sliver > generate --mtls 192.168.0.15 --save ~/file --os Mac

[*] Generating new darwin/amd64 implant binary
[*] Symbol obfuscation is enabled
[*] Build completed in 00:00:37
[*] Implant saved to /Users/muser/~/file
```

Implant should be saved in $HOME/file not $HOME/~/file.

**Comment by rkervella** [comment]
Implemented in v1.6 as part of the client rewrite.

**Comment by duraki** [comment]
Awesome and simple implementation at #550. Thanks a lot @rkervella & @usiegl00 🥳
